cmd/common/helpers.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// CheckForceErr treats error as warning, if --force is provided.
+func CheckForceErr(err interface{}) {
+	if err != nil && IsForce() {
+		fmt.Printf("Warning: %s. Proceeding by force!\n", err)
+		return
+	}
+
+	cobra.CheckErr(err)
```

Comment:
It should also print a hint that you can use `--force` to override. Otherwise people will never discover the option.